### PR TITLE
Issue 185 javier

### PR DESCRIPTION
**Pull request by javerom**
- Added new `MessageBar` class for confirmations in the `bottombar`
- Reorganized start_stop_handler 
- Connected start_stop_handler to messagebar instead of messagebox
- *** Still need to do this for lung recruitment ***

### CODE REVIEW

**Comment by javerom** [comment]
- Implemented confirmation (MessageBar) for lung recruitment

**Comment by carmelopellegrino** [comment]
Hi javier! great job! One of the requirements is that none of the confirm buttons is in the place where the start/stop/CSP buttons was. I hope it's just matter of moving the Yes button on the left :grimacing:

**Comment by carmelopellegrino** [comment]
Another issue: from time to time, when you click on the `CSP` button, the submenu stays there and even clicking on YES and NO do nothing. I don't have any error message in the terminal.
To reproduce: click on `CSP`, then `YES`, then on `Stop Lung recruitment` and repeat. If you are quick enough it stops there. I can think this is something related to the `QTimer` that does the countdown, but didn't checked

**Comment by carmelopellegrino** [comment]
The problem is in lines 44-45 of `messagebar/messagebar.py`. I've added some `print(self.prev_menu)` around `self.bottombar.setCurrentWidget(self)`.

Doing the cycle reported above I read in the terminal:
```
<special.special.SpecialBar object at 0x7faef08b3d30>
<special.special.SpecialBar object at 0x7faef08b3d30>
<special.special.SpecialBar object at 0x7faef08b3d30>
<special.special.SpecialBar object at 0x7faef08b3d30>
<special.special.SpecialBar object at 0x7faef08b3d30>
<special.special.SpecialBar object at 0x7faef08b3d30>
<messagebar.messagebar.MessageBar object at 0x7faef088cee0>
<messagebar.messagebar.MessageBar object at 0x7faef088cee0>
```

**Comment by javerom** [comment]
Ok I've fixed this issue by adding safety to `messagebar`. However, I think the base issue is that (for some reason) the `toggle_lung_recruitment` function is sometime being called twice, which has something to do with the QTimer. The fixe for `messagebar` prevents this being a problem from the confirmation, but this may still be an issue for other things.

File: gui/messagebar/messagebar.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+    def cleanup(self):
+        """
+        Remove cancel and confirm functions for safety
+        """
+        self.func_cancel = None
+        self.func_confirm = None
+
+    def return_menu(self):
+        print(self.prev_menu)
```
Comment (carmelopellegrino) [inline gui/messagebar/messagebar.py:100]: you liked it :)

**Pull request merged**